--- src/Makefile.BASE
+++ src/Makefile.BASE
@@ -114,15 +114,15 @@ copydirs:
 
 install: copydirs
 	@(for i in $(FULLDIR); do echo cd $$i; cd $$i; make MACHINE=$(MACHINE) XINCLUDE="$(XINCLUDE)" SPRNG_LIB=$(SPRNG_LIB) DISKIOSUBDIR="$(DISKIOSUBDIR)" INSTALLDIR="$(INSTALLDIR)" INSTALLBIN="$(INSTALLBIN)" RANLIB="$(RANLIB)" install; cd ..;done)
-	@cp genesis$(EXE_EXT) "$(INSTALLDIR)"
+	@cp genesis$(EXE_EXT) "$(INSTALLDIR)"/bin
 	@echo "Done with full install"
 
 nxinstall: copydirs
 	@(for i in $(NXDIR); do echo cd $$i; cd $$i; make MACHINE=$(MACHINE) SPRNG_LIB=$(SPRNG_LIB) DISKIOSUBDIR="$(DISKIOSUBDIR)" INSTALLDIR=$(INSTALLDIR) INSTALLBIN=$(INSTALLBIN) RANLIB="$(RANLIB)" install; cd ..;done)
-	@cp nxgenesis$(EXE_EXT) "$(INSTALLDIR)"
+	@cp nxgenesis$(EXE_EXT) "$(INSTALLDIR)"/bin
 	@echo "Done with non-X install"
 
 mininstall: copydirs
 	@(for i in $(MINDIR); do echo cd $$i; cd $$i; make MACHINE=$(MACHINE) SPRNG_LIB=$(SPRNG_LIB) INSTALLDIR="$(INSTALLDIR)" INSTALLBIN="$(INSTALLBIN)" RANLIB="$(RANLIB)" install; cd ..;done)
-	@cp mingenesis$(EXE_EXT) "$(INSTALLDIR)"
+	@cp mingenesis$(EXE_EXT) "$(INSTALLDIR)"/bin
 	@echo "Done with minimal install"
